zeppelin-web/src/index.html

```diff
@@ -100,6 +100,20 @@
     <![endif]-->
     <!-- endbuild -->
 
+    <script type="text/x-mathjax-config">
```

Comment:
This is how MathJax is configured. And there's no needs to use this in anywhere else again. So i think it's better to leave it as is.